{"level":"error","message":"deliberate","service":"user-service"}
{"level":"error","message":"deliberate"}
{"level":"error","message":"deliberate"}
{"level":"error","message":"deliberate"}
{"level":"error","message":"","err":{},"timestamp":"2021-03-31T22:31:01.782Z"}
{"level":"error","message":"deliberate","err":{},"timestamp":"2021-03-31T22:31:59.633Z"}
{"message":"deliberate","err":{},"level":"error","timestamp":"2021-03-31T22:32:51.902Z"}
{"message":"deliberate","err":{},"level":"error","timestamp":"2021-03-31T22:38:53.467Z"}
{"message":"deliberate","err":{},"level":"error","timestamp":"2021-03-31T22:42:26.245Z"}
{"message":"deliberate","err":{},"level":"error","timestamp":"2021-03-31T22:42:45.982Z"}
{"message":"deliberate","err":{},"level":"error","timestamp":"2021-03-31T22:43:09.910Z"}
{"message":"deliberate","meta":{},"level":"error","timestamp":"2021-03-31T22:45:23.707Z"}
{"message":"deliberate","meta":"Error: deliberate\n    at E:\\node project\\CompleteP\\routes\\departments.js:34:11\n    at E:\\node project\\CompleteP\\middleware\\asyncHandler.js:5:20\n    at Layer.handle [as handle_request] (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\index.js:284:7\n    at Function.process_params (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (E:\\node project\\CompleteP\\node_modules\\express\\lib\\router\\index.js:275:10)","level":"error","timestamp":"2021-03-31T22:46:19.727Z"}
{"message":"deliberate","ex":{},"level":"error","timestamp":"2021-03-31T23:28:46.873Z"}
{"message":"deliberate","meta":"Error: deliberate\n    at Object.<anonymous> (E:\\node project\\CompleteP\\index.js:42:8)\n    at Module._compile (internal/modules/cjs/loader.js:1156:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)\n    at Module.load (internal/modules/cjs/loader.js:1000:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:899:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)\n    at internal/main/run_main_module.js:18:47","level":"error","timestamp":"2021-03-31T23:29:35.546Z"}
{"message":"deliberate","meta":"Error: deliberate\n    at Object.<anonymous> (E:\\node project\\CompleteP\\index.js:42:8)\n    at Module._compile (internal/modules/cjs/loader.js:1156:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)\n    at Module.load (internal/modules/cjs/loader.js:1000:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:899:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)\n    at internal/main/run_main_module.js:18:47","level":"error","timestamp":"2021-03-31T23:30:10.661Z"}
{"message":"Promise.reject is not a constructor","meta":"TypeError: Promise.reject is not a constructor\n    at Object.<anonymous> (E:\\node project\\CompleteP\\index.js:42:25)\n    at Module._compile (internal/modules/cjs/loader.js:1156:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)\n    at Module.load (internal/modules/cjs/loader.js:1000:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:899:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)\n    at internal/main/run_main_module.js:18:47","level":"error","timestamp":"2021-03-31T23:35:57.831Z"}
{"message":"Promise.reject is not a constructor","meta":"TypeError: Promise.reject is not a constructor\n    at Object.<anonymous> (E:\\node project\\CompleteP\\index.js:42:25)\n    at Module._compile (internal/modules/cjs/loader.js:1156:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)\n    at Module.load (internal/modules/cjs/loader.js:1000:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:899:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)\n    at internal/main/run_main_module.js:18:47","level":"error","timestamp":"2021-03-31T23:36:17.881Z"}
{"message":"Promise.reject is not a constructor","meta":"TypeError: Promise.reject is not a constructor\n    at Object.<anonymous> (E:\\node project\\CompleteP\\index.js:42:25)\n    at Module._compile (internal/modules/cjs/loader.js:1156:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)\n    at Module.load (internal/modules/cjs/loader.js:1000:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:899:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)\n    at internal/main/run_main_module.js:18:47","level":"error","timestamp":"2021-03-31T23:36:50.061Z"}
{"message":"Promise.reject is not a constructor","meta":"TypeError: Promise.reject is not a constructor\n    at Object.<anonymous> (E:\\node project\\CompleteP\\index.js:42:25)\n    at Module._compile (internal/modules/cjs/loader.js:1156:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)\n    at Module.load (internal/modules/cjs/loader.js:1000:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:899:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)\n    at internal/main/run_main_module.js:18:47","level":"error","timestamp":"2021-03-31T23:37:02.615Z"}
{"message":"Promise rejected","meta":"Error: Promise rejected\n    at Object.<anonymous> (E:\\node project\\CompleteP\\index.js:53:6)\n    at Module._compile (internal/modules/cjs/loader.js:1156:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)\n    at Module.load (internal/modules/cjs/loader.js:1000:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:899:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)\n    at internal/main/run_main_module.js:18:47","level":"error","timestamp":"2021-04-01T00:20:20.677Z"}
